src/components/Panels/Guider/Diagram/Diagram.tsx

```diff
@@ -117,10 +119,40 @@ function Flow() {
       });
     }
 
+    // Create node if not present
+    const wfsFromDevices = state.m2TipTiltSource.split(',');
+    if (state.m2FocusSource && state.m2FocusSource in WFS_LIST && !(state.m2FocusSource in wfsFromDevices)) {
+      wfsFromDevices.push(state.m2FocusSource);
+    }
+    if (
+      state.m2ComaM1CorrectionsSource &&
+      state.m2ComaM1CorrectionsSource in WFS_LIST &&
```

Comment:
You should use `WFS_LIST.includes`. The `in` keyword checks if the property exists on an object (for example, `'a' in {a:1} === true`